src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,44 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int length = 10;
```

Comment:
```suggestion
    private static final int MAX_STORAGE_SIZE = 10;
```

what about constant naming?